dcos_launch/onprem.py

```diff
@@ -166,70 +164,31 @@ def _fault_domain_helper(self) -> str:
-            # already been checked
-            if host == cluster.bootstrap_host:
-                continue
-            host_ssh_client.wait_for_ssh_connection(host.public_ip, self.config['ssh_port'])
-        # print the IPs for use in live-debugging the installation
-        log.info('Cluster master IP(s): {}'.format(cluster.masters))
-        log.info('Cluster public agent IP(s): {}'.format(cluster.public_agents))
-        log.info('Cluster private agent IP(s): {}'.format(cluster.private_agents))
-        log.info('Cluster bootstrap IP: {}'.format(cluster.bootstrap_host))
-        try:
-            self.get_ssh_client(user='bootstrap_ssh_user').command(self.bootstrap_host, ['test', '-f', STATE_FILE])
-            last_complete = self.get_last_state()
-            log.info('Detected previous launch state, continuing '
-                     'from last complete stage ({})'.format(last_complete))
-        except subprocess.CalledProcessError:
-            log.info('No installation state file detected; beginning fresh install...')
-            last_complete = None
-
-        if last_complete is None:
-            cluster.setup_installer_server(self.config['installer_url'], False)
-            last_complete = 'SETUP'
-            self.post_state(last_complete)
-
-        installer = dcos_test_utils.onprem.DcosInstallerApiSession(Url(
-            'http', self.bootstrap_host, '', '', '', self.config['installer_port']))
-        if last_complete == 'SETUP':
-            last_complete = 'GENCONF'
-            installer.genconf(self.get_completed_onprem_config(cluster))
-            self.post_state(last_complete)
-        if last_complete == 'GENCONF':
-            installer.preflight()
-            last_complete = 'PREFLIGHT'
-            self.post_state(last_complete)
-        if last_complete == 'PREFLIGHT':
-            installer.deploy()
-            last_complete = 'DEPLOY'
-            self.post_state(last_complete)
-        if last_complete == 'DEPLOY':
-            installer.postflight()
-            last_complete = 'POSTFLIGHT'
-            self.post_state(last_complete)
-        if last_complete != 'POSTFLIGHT':
-            raise dcos_launch.util.LauncherError('InconsistentState', 'State on bootstrap host is: ' + last_complete)
+        bootstrap_host = cluster.bootstrap_host.public_ip
+        self.bootstrap_ssh_client.wait_for_ssh_connection(bootstrap_host)
+        with self.bootstrap_ssh_client.tunnel(bootstrap_host) as t:
+            installer_path = platforms_onprem.prepare_bootstrap(t, self.config['installer_url'])
+            complete_config = self.get_completed_onprem_config()
+            platforms_onprem.do_genconf(t, complete_config, installer_path)
+        platforms_onprem.install_dcos(
+            cluster,
+            self.get_ssh_client(),
+            self.config['prereqs_script_filename'] if self.config['install_prereqs'] else None,
+            complete_config['bootstrap_url'] + ':80/dcos_install.sh',
```

Comment:
Is there a reason the `:80` here wasn't included in the `bootstrap_url`? We should only need to add URIs to `bootstrap_url` to create artifact URLs.

Reply:
Really doesnt need to exist. Can remove